test/ng/directive/inputSpec.js

```diff
@@ -657,17 +657,20 @@ describe('input', function() {
       expect($rootScope.form.alias.$error.month).toBeTruthy();
     });
 
-    it('should allow four or more digits in year', function() {
-      var inputElm = helper.compileInput('<input type="month" ng-model="value"  ng-model-options="{timezone: \'UTC\'}"/>');
 
-      helper.changeInputValueTo('10123-03');
```

Comment:
@Narretz when you were preparing this change, were you able to determine the root cause of these inputSpec failures?  I've been looking at these failures recently as part of the work @jdalton mentioned to include AngularJS tests in our Edge Regression testing.  I have concluded that this line `helper.changeInputValueTo('10123-03');` is failing to set the `.value` property on the HTML element in Edge, which is what results in the eventual failure, but I had to stop before getting to "why" the value property isn't getting set. 

If there is an Edge bug behind this I'd love to make sure it gets filed.